### PR TITLE
Tools: add airspeed defaults to vehicle defaults files

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

**Review by tridge — APPROVED** [review]

File: Tools/autotest/default_params/blimp.parm [hunk 1]
```diff
@@ -1,3 +1,6 @@
+ARSPD_PIN 1
+ARSPD_BUS 2
+ARSPD_TYPE 0
```
Comment (rmackay9) [inline Tools/autotest/default_params/blimp.parm:3]: remove the ARSPD_TYPE

File: Tools/autotest/default_params/copter.parm [hunk 2]
```diff
@@ -1,5 +1,8 @@
 FRAME_TYPE	0
 FS_THR_ENABLE   1
+ARSPD_PIN 1
+ARSPD_BUS 2
+ARSPD_TYPE 0
```
Comment (hendjoshsr71) [inline Tools/autotest/default_params/copter.parm:5]: remove the types so that it matches as close as possible to a real vehicle. 

```suggestion
```

**Review by hendjoshsr71 — CHANGES_REQUESTED** [review]
Thanks Henry!

File: Tools/autotest/default_params/sailboat.parm [hunk 3]
```diff
@@ -1,4 +1,7 @@
 ARSPD_OFFSET 2013
+ARSPD_PIN 1
+ARSPD_BUS 2
+ARSPD_TYPE 0
```
Comment (rmackay9) [inline Tools/autotest/default_params/sailboat.parm:4]: @Hwurzburg one more here too..

Comment (Hwurzburg) [reply Tools/autotest/default_params/sailboat.parm]: sorry...got it

**Review by hendjoshsr71 — APPROVED** [review]
lgtm

**Review by tridge — APPROVED** [review]

**Pull request merged**